wp-cli/vip-two-factor.php

```diff
@@ -1,35 +1,130 @@
 <?php
 
 class VIP_Two_Factor_Command extends WPCOM_VIP_CLI_Command {
+	/**
+	 * Reports on the 2FA status for users of a site. Lists all users by default.
+	 * Note: Users without `manage_options` cap will be listed as "n/a" for 2FA status.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--2fa-enabled=<true|false>]
+	 * : Filter by whether 2FA is enabled or not.
+	 *
+	 * [--role=<string>]
+	 * : Filter by user role.
+	 *
+	 * [--2fa-provider=<string>]
+	 * : Filter by 2FA provider. Accepts values of: email, totp, fido_u2f, backup_codes, dummy
+	 *
+	 * [--user_login=<login|id|email>]
+	 * : Filter by user login, ID, or email.
+	 *
+	 *
+	 * ## EXAMPLES
+	 *
+	 *      wp vip two-factor report
+	 *      wp vip two-factor report --2fa-enabled=true
+	 *      wp vip two-factor report --2fa-provider=email
+	 *      wp vip two-factor report --role=administrator
+	 *      wp vip two-factor report --role=administrator --2fa-enabled=false
+	 *      wp vip two-factor report --role=administrator --2fa-enabled=true --2fa-provider=email
+	 *      wp vip two-factor report --user_login=wpvip
```

Comment:
```suggestion
	 *     # List 2FA status for all users.
	 *     $ wp vip two-factor report
	 *
	 *     # List users with 2FA enabled. 
	 *     $ wp vip two-factor report --2fa-enabled=true
	 *
	 *     # List users who use email as their authentication factor. 
	 *     $ wp vip two-factor report --2fa-provider=email
	 *
	 *     # List 2FA status for administrators. 
	 *     $ wp vip two-factor report --role=administrator
	 *
	 *     # List administrators who don't have 2FA enabled. 
	 *     $ wp vip two-factor report --role=administrator --2fa-enabled=false
	 *
	 *     # List 2FA status for administrators using the email authentication factor. 
	 *     $ wp vip two-factor report --role=administrator --2fa-enabled=true --2fa-provider=email
	 *
	 *     # List 2FA status only for the wpvip user. 
	 *     $ wp vip two-factor report --user_login=wpvip
```

Question:
> List users who use email as their authentication factor. 

Is this just primary factor, or does it include secondary providers as well?

Reply:
It includes secondary providers 🙂